### ISSUE TITLE
Animation Chapter

### ISSUE THREAD

**Issue by bebraw** [survivejs/webpack_react#166]

**Comment by jvanzyl** [comment]
Are these just ideas for bonus chapters? Or chapters available for people who purchased the book? The book is excellent and I'm looking at animation next so if there is material that is fantastic.

**Comment by bebraw** [comment]
Yeah, they are development ideas at this point. Now I'm pushing towards a wider scale release with the current content. Ideally I'll get a paper version done. A lot depends on how this bigger release (2.0) goes. I'll be blogging a lot regardless.

I would suggest starting from [react-motion](https://github.com/chenglou/react-motion). See also the [official docs](https://facebook.github.io/react/docs/animation.html) and perhaps [react-animate](https://github.com/elierotenberg/react-animate).

**Comment by jvanzyl** [comment]
Great, thanks for the suggestions!

**Comment by bebraw** [comment]
@jvanzyl One more. [velocity-react](https://github.com/twitter-fabric/velocity-react) just got [announced](https://fabric.io/blog/introducing-the-velocityreact-library).

One interesting question is that is it possible to integrate **velocity-react** and **React DnD** together somehow (i.e., in Kanban ordering).

**Comment by omidfi** [comment]
this talk was interesting in react rally: 
[animated](https://www.youtube.com/watch?v=xtqUJVqpKNo) by Christopher Chedeau

**Comment by karlnp** [comment]
We're using GSAP with React, which has been interesting to figure out. It may be a good idea to delve into the workings of something like react-motion - in other words, how does web animation with React differ from web animation without React? Let me know if you'd like an email summarizing what our challenges were. (Have to check and see how much I'm allowed to share from the work case, but I can speak easily in the general case.)

**Comment by bebraw** [comment]
@karlnp Sure. If you can drop me a mail, that would be great! I feel we're still figuring this area of React out.
